lib/css/components/links.less

```diff
@@ -1,214 +1,102 @@
 a,
```

Comment:
What about renaming this file to `link.less` (singular) now that it only contains `s-link` related styles?

Reply:
Good call! Consider it done!